src/components/bcros/todo/List.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <BcrosTodoItem
+    v-for="todoItem in todos"
+    :key="todoItem.uiUuid"
+    :item="todoItem"
+    :title="todoItem.title"
+    data-cy="todoItemList"
```

Comment:
should have a class for a divider in between each item (i.e. `divide-y-1` -- something like that I think)